### ISSUE TITLE
Fractional percentage notation in color parser

### ISSUE THREAD

**Issue by kimikage** [JuliaGraphics/Colors.jl#431]
The color parser has only accepted integer percentages since Color.jl was separated from Compose.jl.(cf. 67384015e505a472014342c9fec4e3ad100a3ba6)

```julia
julia> colorant"rgb(100%,50%,0%)"
RGB{N0f8}(1.0,0.502,0.0)

julia> colorant"rgb(100%,50%,3.14%)"
ERROR: LoadError: Unknown color: rgb(100%,50%,3.14%)
```

This is practical enough as long as the users write the percentages with `@colorant_str`. So I added the specification to the document in PR #406.

However, other programs may generate a fractional decimal percentage.For example, https://github.com/JuliaLang/julia-logo-graphics/blame/168fb6c1164e341df360ed6ced519e1e0cb7de3a/images/julia-logo-color.svg.

I do never intend to make the parser fully CSS compliant, but I'd like to remove the restriction.

**Comment by kimikage** [comment]
Julia v1.5 will be released soon, so I'd like to "cancel" the deprecation warning.
https://github.com/JuliaGraphics/Colors.jl/blob/53c2acb9c4ac4aa16f9de036a1a96b36a86ce924/src/parse.jl#L143-L147

Instead, in Colors v0.13 I'll add a deprecation warning to:
https://github.com/JuliaGraphics/Colors.jl/blob/53c2acb9c4ac4aa16f9de036a1a96b36a86ce924/src/parse.jl#L212
(cf. https://github.com/JuliaGraphics/Colors.jl/pull/406#discussion_r386097241)

**Comment by timholy** [comment]
So by "cancel," do you mean "turn the warning into an error" or something else?

I agree with adding a depwarn to that method in Colors. But there's no conservation of depwarns, you can have as many as you want :smile:.

**Comment by kimikage** [comment]
I'm thinking about purely removing the five lines of `depwarn` above, leaving [the note](https://github.com/JuliaGraphics/Colors.jl/blob/53c2acb9c4ac4aa16f9de036a1a96b36a86ce924/src/parse.jl#L192-L194) in the docstring. (cf. https://github.com/JuliaGraphics/Colors.jl/issues/419)

**Comment by kimikage** [comment]
> you can have as many as you want 😄.

Yes. I think FixedPointNumbers/ColorTypes/ColorVectorSpace are starting to move towards the next minor (not patch) level updates. I'm just trying to enumerate the (trivial) fixes to piggyback on the update.
